### DISCUSSION TITLE
Overworked developer fails when adding @repo/ prefix to lib name.

### DISCUSSION THREAD

**Discussion by mwawrusch** [tailwindlabs/tailwindcss#13512] [Help] [answered]
I use tailwind in a turborepo and compile the css in the top level project. To this extent i load all the imported packages like so:
```
const allRojiFolders = fs.readdirSync('../../node_modules')
  .filter(folder => folder.startsWith('roji-ui') || folder.startsWith('roji-domain'))
  .map(folder => `../../node_modules/${folder}/src/**/*.{js,ts,jsx,tsx}`);
```
This results in a list of all src files that need to be ingested by tailwind.

```
module.exports = {
  ...tailwindConfig,
  content: [
    './src/**/*.{js,ts,jsx,tsx}',
    ...allRojiFolders,
  ],
}
```

```
"name": "roji-domain-site-header" ==> "name": "@repo/roji-domain-site-header"
```

All was good and dandy till I started to change the package names from my packages from 'mylib' to '@repo/mylib' as suggested by the latest turbo. 

This resulted in tailwind no longer creating css for the newly renamed library. Is this a tailwind bug?

**Comment by adamwathan** [comment, marked as answer]
Is the problem that this code needs to be updated to look for `@repo` before the package name?

```
const allRojiFolders = fs.readdirSync('../../node_modules')
  .filter(folder => folder.startsWith('roji-ui') || folder.startsWith('roji-domain'))
  .map(folder => `../../node_modules/${folder}/src/**/*.{js,ts,jsx,tsx}`);
```

If you can provide a reproduction we can look.

**Reply by mwawrusch** [reply]
Haha, OMG, dumbest mistake in a long time. I was gobbling the node_modules, where the files are stored under the @repo/ subfolder.
Really appreciate your quick feedback.
